### DISCUSSION TITLE
Dynamic baseBranches names

### DISCUSSION THREAD

**Discussion by aaronware** [renovatebot/renovate#14103] [Request Help]
Our team does a lot of maintenance. Some of that maintenance is done ad-hoc for patches and things of that nature. But some of it is scheduled (each month). I couldn't find anything in the documentation, but it would be great to have the ability to dynamically name a branch in our scenario it would be something like `maintenance/FEB-2022` as a simple example.

Is that something that is possible, I was thinking with some sort of macro/merge tag type functionality.

**Comment by rarkins** [comment]
A feature request exists for this, but is missing a reproduction in order to reach `status:ready` state: https://github.com/renovatebot/renovate/issues/5027

**Reply by aaronware** [reply]
Just putting this here https://github.com/renovatebot/renovate/blob/main/docs/development/minimal-reproductions.md so I can circle back to it. I should be able to spin up an example
